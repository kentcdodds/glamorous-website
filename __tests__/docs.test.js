/* global it, expect, describe */

import React from 'react'
import renderer from 'react-test-renderer'
import Docs from '../pages/docs'

describe('Testing Docs', () => {
  it('Renders!', () => {
    const component = renderer.create(<Docs/>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
