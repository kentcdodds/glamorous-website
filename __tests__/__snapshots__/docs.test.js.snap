// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Docs Renders! 1`] = `
.css-204t0b,
[data-css-204t0b] {
  font-family: "Montserrat", sans-serif;
  background-color: #FFEFEF;
}

.css-204t0b:after,
[data-css-204t0b]:after {
  position: absolute;
  content: "";
  background-image: url(/static/images/g-background.svg);
  background-size: 800px;
  background-repeat: no-repeat;
  background-position: 70% -300px;
  pointer-events: none;
  z-index: 0;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
}

.css-4cv6bo,
[data-css-4cv6bo] {
  position: relative;
  z-index: 1;
}

.css-dkzezy,
[data-css-dkzezy] {
  -webkit-box-align: center;
  -webkit-justify-content: center;
  -webkit-align-items: center;
  -ms-flex-pack: center;
  -webkit-box-pack: center;
  -ms-flex-align: center;
  padding: 0;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  height: 4rem;
  list-style: none;
  justify-content: center;
  align-items: center;
  font-size: 1.25em;
  margin: 0 auto;
  max-width: 50rem;
}

.css-1oaxt76,
[data-css-1oaxt76] {
  padding-left: 10px;
  padding-right: 10px;
  padding-bottom: 4px;
}

.css-1xw95l9,
[data-css-1xw95l9] {
  cursor: pointer;
  text-decoration: none;
  color: #F67982;
}

.css-1xw95l9:visited,
[data-css-1xw95l9]:visited {
  color: #BD3D90;
}

.css-1szh0dv,
[data-css-1szh0dv] {
  margin: 20px auto;
  width: 100%;
  max-width: 50rem;
  padding: 0 1rem;
}

.css-1szh0dv img,
[data-css-1szh0dv] img {
  width: 100%;
}

.css-1szh0dv pre,
[data-css-1szh0dv] pre,
.css-1szh0dv code,
[data-css-1szh0dv] code {
  background-color: #eee;
  font-size: 1rem;
}

.css-1szh0dv pre,
[data-css-1szh0dv] pre {
  padding: 1rem;
  overflow-x: scroll;
}

.css-1szh0dv a,
[data-css-1szh0dv] a {
  text-decoration: none;
  color: #ED5C70;
}

.css-1szh0dv p,
[data-css-1szh0dv] p,
.css-1szh0dv li,
[data-css-1szh0dv] li {
  line-height: 2rem;
}

.css-1szh0dv p code,
[data-css-1szh0dv] p code,
.css-1szh0dv li code,
[data-css-1szh0dv] li code,
.css-1szh0dv blockquote code,
[data-css-1szh0dv] blockquote code {
  padding: .3rem;
  border-radius: .3rem;
}

.css-1szh0dv blockquote,
[data-css-1szh0dv] blockquote {
  margin: 1rem 0;
  padding: 0 1rem;
  color: #666;
  border-left: 6px solid #ddd;
}

.css-1djfccv,
[data-css-1djfccv] {
  height: 7px;
  background: -webkit-linear-gradient(to right, #f67982 0%,#f67982 33%,#FEBBC0 33%,#FEBBC0 66%,#FECDCD 66%,#FECDCD 100%);
  background: -moz-linear-gradient(to right, #f67982 0%,#f67982 33%,#FEBBC0 33%,#FEBBC0 66%,#FECDCD 66%,#FECDCD 100%);
  background: linear-gradient(to right, #f67982 0%,#f67982 33%,#FEBBC0 33%,#FEBBC0 66%,#FECDCD 66%,#FECDCD 100%);
}

.css-qyz8mk,
[data-css-qyz8mk] {
  padding-top: 10px;
  padding-bottom: 10px;
  font-size: 0.6em;
  text-align: center;
  background: #fff;
}

.css-1sbv75w,
[data-css-1sbv75w] {
  cursor: pointer;
  text-decoration: underline;
  color: #ED5C70;
}

.css-1sbv75w:visited,
[data-css-1sbv75w]:visited {
  color: #BD3D90;
}

@media only screen and (min-width: 481px) {
  .css-dkzezy,
  [data-css-dkzezy] {
    -webkit-justify-content: flex-end;
    -ms-flex-pack: end;
    -webkit-box-pack: end;
    justify-content: flex-end;
  }
}

<div
  className="css-204t0b"
>
  <div
    className="css-4cv6bo"
  >
    <nav
      className="css-nil"
    >
      <ul
        className="css-dkzezy"
      >
        <li
          className="css-1oaxt76"
        >
          <a
            className="css-1xw95l9"
            onClick={[Function]}
          >
            <svg
              enableBackground="new 0 0 64 64"
              viewBox="0 0 64 64"
              width={20}
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="#ED5C70"
              >
                <path
                  d="M20.8 6.4 40.9 26.5 29.7 37.7 2.2 10.2z"
                />
                <path
                  d="m10.6 5.8c-5.2.9-9 2.7-8.6 4.1.5 1.4 5.1 1.8 10.4.9 5.2-1 9.1-2.8 8.6-4.2-.5-1.4-5.2-1.7-10.4-.8"
                />
              </g>
              <path
                d="m10.8 5.8c-4.9.8-8.5 2.6-8 3.9.5 1.3 4.8 1.6 9.7.8 4.9-.8 8.5-2.6 8-3.8-.5-1.4-4.8-1.8-9.7-.9"
                fill="#ff8080"
              />
              <path
                d="m22.6 16.5h18.2v23.8h-18.2z"
                fill="#e8e8e8"
                transform="matrix(.7071-.7071.7071.7071-10.7928 30.6765)"
              />
              <path
                d="m43.7 47h19v5.9h-19z"
                fill="#555b61"
                transform="matrix(.707-.7072.7072.707-19.7271 52.2523)"
              />
              <path
                d="m27.2 30.7h19v5.5h-19z"
                fill="#b2c1c0"
                transform="matrix(.7069-.7073.7073.7069-12.8952 35.7644)"
              />
              <path
                d="m34.8 29.8h21v24.4h-21z"
                fill="#3e4347"
                transform="matrix(.7071-.7071.7071.7071-16.45 44.3334)"
              />
            </svg>
          </a>
        </li>
        <li
          className="css-1oaxt76"
        >
          <a
            className="css-1xw95l9"
            onClick={[Function]}
          >
            Guides
          </a>
        </li>
        <li
          className="css-1oaxt76"
        >
          <a
            className="css-1xw95l9"
            onClick={[Function]}
          >
            Examples
          </a>
        </li>
        <li
          className="css-1oaxt76"
        >
          <a
            className="css-1xw95l9"
            onClick={[Function]}
          >
            Documentation
          </a>
        </li>
      </ul>
    </nav>
    <div
      className="css-1szh0dv"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "",
          }
        }
      />
    </div>
    <div>
      <div
        className="css-1djfccv"
      />
      <footer
        className="css-qyz8mk"
      >
        Made with 
        <span
          aria-label="love"
          role="img"
        >
          ðŸ’™
        </span>
         by
         
        <a
          className="css-1sbv75w"
          href="https://github.com/paypal"
        >
          PayPal
        </a>
        .
      </footer>
    </div>
  </div>
</div>
`;
